tox.ini

```diff
@@ -47,8 +47,8 @@ basepython = python2.7
 deps =
     docutils
 commands =
-    {envbindir}/rst2html.py --report=info --halt=warning README.rst /dev/null
-    {envbindir}/rst2html.py --report=info --halt=warning CHANGES.rst /dev/null
+    {envbindir}/rst2html.py --syntax-highlight=short --report=info --halt=warning README.rst /dev/null
```

Comment:
According to travis, seems like this isn't valid in Python 2.7?

Reply:
Yes, even without `--syntax-highlight=short` (see previous Travis build) 🙁